apispec/core.py

```diff
@@ -106,11 +113,22 @@ def to_dict(self):
             'swagger': SWAGGER_VERSION,
             'info': self.info,
             'definitions': self._definitions,
+            'parameters': self._parameters,
             'paths': self._paths,
         }
         ret.update(self.options)
         return ret
 
+    def add_parameter(self, _name, _in, **kwargs):
```

Comment:
Have tried to address this in latest commit. Have replaced `in` for `location`, which is consistent with the swagger spec.

The reason I pulled from kwargs was because the spec for parameters is relatively [complex](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#parameter-object). I suspect it wouldn't make sense to validate whether the caller is constructing a valid parameter at this point, since it would be possible for a parameter to be defined through alternat code paths.